Winium/Winium.sln

```diff
@@ -55,58 +57,169 @@ Global
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|ARM = Debug|ARM
```

Comment:
I don't think we need that much configurations. Visual Studio creates them automatically when any project is added. Need to investigate if that can be prevented in future. Meanwhile, simple deletion of unnecessary configurations will suffice.